test/integration/targets/vmware_object_rename/aliases

```diff
@@ -0,0 +1,4 @@
+cloud/vcenter
+unsupported
```

Comment:
I don't think we need the `unsupported` alias. The test-suite should be able to run on a single vcenter.